ocs_ci/ocs/resources/ocs.py

```diff
@@ -97,11 +97,12 @@ def create(self, do_reload=True):
         return status
 
     def delete(self, wait=True, force=False):
-        result = self.ocp.delete(
-            resource_name=self.name, wait=wait, force=force
-        )
-        self._is_deleted = True
-        return result
+        if not self.is_deleted:
+            result = self.ocp.delete(
+                resource_name=self.name, wait=wait, force=force
+            )
+            self._is_deleted = True
+            return result
```

Comment:
Now you are returning boolean value only when it's not deleted otherwise the None is returned as default.  I looked to this issue yesterday and had some commit which I didn't push finally cause of I switched to something else.

```python
    def delete(self, wait=True, force=False):
        if self._is_deleted:
            log.warning(
                f"Attempt to remove resource: {self.name} which is"
                f"already deleted! Skipping delete of this resource!"
            )
            result = True
        else:
            result = self.ocp.delete(
                resource_name=self.name, wait=wait, force=force
            )
            self._is_deleted = True
        return result
```
We can have info level instead of warning bus we should return True if it's already deleted I think.

Maybe if you are touching this function, it's worth to add docstring. Thanks.

Reply:
will fix the return and docstrings too. Thanks